src/alpaka/AlpakaCore/deviceAllocatorStatus.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HeterogeneousCore_AlpakaUtilities_deviceAllocatorStatus_h
+#define HeterogeneousCore_AlpakaUtilities_deviceAllocatorStatus_h
+
+#include <map>
+
+#include "AlpakaCore/alpakaConfig.h"
+
+namespace cms {
+  namespace alpakatools {
+    namespace allocator {
+      struct TotalBytes {
+        size_t free;
+        size_t live;
+        size_t liveRequested;  // CMS: monitor also requested amount
+        TotalBytes() { free = live = liveRequested = 0; }
+      };
+
+      inline int getIdxOfDev(const ::ALPAKA_ACCELERATOR_NAMESPACE::Device& device) {
+        static const auto devices{alpaka::getDevs<::ALPAKA_ACCELERATOR_NAMESPACE::Platform>()};
+        return (std::find(devices.begin(), devices.end(), device) - devices.begin());
+      }
```

Comment:
(just repeating) Given https://github.com/cms-patatrack/pixeltrack-standalone/blob/master/src/alpaka/AlpakaCore/getDevIndex.h this is not needed.